pkg/apiserver/apiserver_test.go

```diff
@@ -201,7 +204,7 @@ func handleLinker(storage map[string]rest.Storage, selfLinker runtime.SelfLinker
 }
 
 func newTestRequestInfoResolver() *RequestInfoResolver {
-	return &RequestInfoResolver{sets.NewString("api", "apis"), sets.NewString("api")}
+	return &RequestInfoResolver{sets.NewString("api", "apis"), sets.NewString()}
```

Comment:
If we're testing api versions with groups, we should test that under `apis/`

I also really don't want to lose the groupless tests... can we change "api/version" to "api/v1" in this file so we're actually testing a groupless API that will parse into a GroupVersion correctly, and duplicate the test cases for an API with groups?